REFERENCE.md

```diff
@@ -2495,6 +2526,48 @@ Possible strings for ExecStart, ExecStartPrep, ...
 
 Alias of `Variant[Enum[''], Pattern[/^[@\-:]*(\+|!|!!)?[@\-:]*\/.*/], Pattern[/^[@\-:]*(\+|!|!!)?[@\-:]*[^\/]*(\s.*)?$/]]`
 
+### <a name="Systemd--Unit--Slice"></a>`Systemd::Unit::Slice`
+
+Possible keys for the [Slice] section of a unit file
+
+* **See also**
+  * https://www.freedesktop.org/software/systemd/man/systemd.slice.html
+  * https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html
+
+Alias of
+
+```puppet
+Struct[{
+    Optional['CPUAccounting']       => Boolean,
+    Optional['CPUQuota']            => Pattern['^([1-9][0-9]*)%$'],
+    Optional['CPUShares']           => Integer[2,262144],
```

Comment:
`CPUShares` and a couple of others are deprecated.
https://www.freedesktop.org/software/systemd/man/latest/systemd.resource-control.html#systemd%20252

Of cause there is still systems which do not provide cgroups v2 yet, but for a new feature I wonder if it is worth implementing support for those.

Reply:
> `CPUShares` and a couple of others are deprecated.
> https://www.freedesktop.org/software/systemd/man/latest/systemd.resource-control.html#systemd%20252
> 
> Of cause there is still systems which do not provide cgroups v2 yet, but for a new feature I wonder if it is worth implementing support for those.

Think we need the  the older ones, we are still using them for sure on CentOS 7 and even EL8 is cgroups v1 by default.